index.html

```diff
@@ -289,6 +289,16 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <div class="contact">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="POST" class="contact-form">
```

Comment:
Unfortunately, your form is not sending information to Formspree, please add the required action endpoint URL from Formspree.